src/DataFrames.jl

```diff
@@ -75,7 +75,61 @@ export @~,
        stackdf,
        unique!,
        unstack,
-       writetable
+       writetable,
+       # FIXME: unexport, these should go in Base or nowhere
```

Comment:
Why would they be limited to data frames? In R, these functions work on any vector, returning the first/last elements. There's been some discussion in Base about that, a possible solution was to extend `first` and `last`.

It shouldn't be related to this PR in theory, but these methods are defined in DataArrays, that's why I had to do something about them. But I can keep them in DataFrames for now and tackle that problem later.